### DISCUSSION TITLE
How to show scrollbar in toolbar with 2.4.2

### DISCUSSION THREAD

**Discussion by zhhanging** [epam/ketcher#1564] [Q&A]
we used to get a scrollbar in left/right toolbar if the toolbar is too high.
![CleanShot 2022-05-05 at 15 44 52@2x](https://user-images.githubusercontent.com/39266588/166881358-f3049c87-9cc1-44cf-a8f8-4acebae04140.png)
but now we only get a small triangle click button for that, and the button seems not working in safari. Is there any workarounds to get a scrollbar at the toolbar? Or is there is a way to make it easier to navigate the toolbar?
![CleanShot 2022-05-05 at 15 47 15@2x](https://user-images.githubusercontent.com/39266588/166881643-5199de7d-7d43-4dc6-ba50-5086d112c04a.png)

**Comment by AndreiMazol** [comment]
Dear @zhhanging ,

scrollbar at the left toolbar  has been hidden. We will check if triangle buttons work in Safari. The issue will  be fixed as a part of the nearest release (2.5).

Regards,
Ketcher team
